README
NAME

    Task::BeLike::RSRCHBOY - Modules RSRCHBOY uses!

VERSION

    version 0.006

TASK CONTENTS

 CPAN/GitHub Tools

  local::lib

  Pod::Cpandoc

  App::cpanminus::reporter

  App::cpanoutdated

  App::cpanminus::reporter

  App::gh

  CPAN::Mini

  CPAN::Mini::Inject

  Task::CPAN::Reporter

  Pod::Cpandoc

 DB bits

  App::AltSQL

  DBD::SQLite 1.31

  DBIx::Class 0.08192

  DBIx::Class::Schema::Loader 0.07010

  DBIx::Class::Migration

 Date/Time handling

  DateTime

  DateTime::Format::DB2

  DateTime::Format::DBI

  DateTime::Format::MySQL

  DateTime::Format::Pg

 Hygiene

  common::sense

  namespace::autoclean

  strictures

  Perl::Tidy

  Perl::Critic

 Class bits

  Class::Method::Modifiers::Fast

  Class::Method::Modifiers

  Reindeer 0.016

  MooseX::App::Cmd

  MooseX::AutoDestruct 0.007

  MooseX::RelatedClasses

  MooseX::Types::Perl

  Moo

  MooX::Types::MooseLike

  MooX::Cmd

 Testing

  Directory::Scratch

  TAP::Harness::Restricted

  Test::Moose::More 0.017

  Test::Routine 0.015

  Test::TempDir::Tiny

 autobox

  autodie 2.22

  autobox::Core

  autobox::dump

  autobox::JSON

  autobox::Colors

 Other Libraries I Use

  aliased

  lib::if::dev

  opts 0.05

  App::Cmd

  Capture::Tiny

  Carp::Always::Color

  Config::JFDI

  Carp::Always

  File::chdir

  Path::Class

  Smart::Args

  Smart::Comments

  Sub::Install

  Sub::Exporter

  Try::Tiny

 Helpers

  Regexp::Debugger

  L

  V

 CPAN-ish Development

  Devel::Confess

  Dist::Zilla::App::Command::issues

  Dist::Zilla::PluginBundle::RSRCHBOY

SEE ALSO

    Dist::Zilla::PluginBundle::RSRCHBOY

    Task::BeLike::RSRCHBOY::Web

AUTHOR

    Chris Weyl <cweyl@alumni.drew.edu>

COPYRIGHT AND LICENSE

    This software is Copyright (c) 2011 by Chris Weyl.

    This is free software, licensed under:

      The GNU Lesser General Public License, Version 2.1, February 1999



Changes
Revision history for Task-BeLike-RSRCHBOY

{{$NEXT}}

0.007     2015-04-09 20:46:43 PDT-0700
	* + Dist::Zilla::App::Command::cover
	* + Dist::Zilla::App::Command::gh
	* + Dist::Zilla::App::Command::dumpwith
	* = namespace::autoclean >= 0.24
	* - Directory::Scratch
	* + Path::Tiny
	* + MooseX::Types::LoadableClass
	* + MooseX::AttributeShortcuts
	* + MooseX::AlwaysCoerce
	* + OrePAN2

0.006     2015-04-06 20:44:02 PDT-0700
	* + App::Cmd
	* + Devel::Confess
	* + Dist::Zilla::App::Command::issues
	* + Dist::Zilla::PluginBundle::RSRCHBOY
	* + L
	* - Git::CPAN::Patch (BackPAN::Index is broken, and we have the gitpan now)
	* - File::Slurp
	* long overdue additions, deletions

0.005     2013-12-06 19:09:34 PST-0800
	* add: App::cpanminus::reporter
	* require autodie >= 2.22; add autobox::Colors
	* add: Carp::Always
	* add: Moo, MooX::Types::MooseLike, MooX::Cmd
	* drop Moose::Autobox, as I'm more an autobox::Core kinda person
	* add App::AltSQL, as it ROCKS.
	* add TAP::Harness::Restricted, Class::Method::Modifiers::Fast
	* drop Role::Basic
	* add Task::CPAN::Reporter
	* add Travis-CI configuration

0.004     2013-02-23 10:23:10 PST-0800
	* add cpandoc, autodie 0.16
	* add MXRC, bump TMM and Reindeer req versions to latest
	* add Directory::Scratch and Test::TempDir
	* add Regexp::Debugger
	* add various autobox packages
	* make Changes more changelogy

0.003     2012-05-29 14:51:31 America/Los_Angeles
	* misc version updates; add Software::License::CCpack

0.002     2012-03-30 20:45:53 America/Los_Angeles
	* Split web deps out

0.001     2011-09-29 11:23:39 America/Los_Angeles
	* initial release


lib/Task/BeLike/RSRCHBOY.pm
package Task::BeLike::RSRCHBOY;

# ABSTRACT: Modules RSRCHBOY uses!

!!42;

__END__

=pkgroup CPAN/GitHub Tools

=pkg local::lib

=pkg App::cpanminus::reporter

=pkg App::cpanoutdated

=pkg App::gh

=pkg CPAN::Mini

=pkg CPAN::Mini::Inject

=pkg OrePAN2

=pkg Pod::Cpandoc

=pkg Task::CPAN::Reporter


=pkgroup DB bits

=pkg App::AltSQL

=pkg DBD::SQLite 1.31

=pkg DBIx::Class 0.08192

=pkg DBIx::Class::Schema::Loader 0.07010

=pkg DBIx::Class::Migration


=pkgroup Date/Time handling

=pkg DateTime

=pkg DateTime::Format::DB2

=pkg DateTime::Format::DBI

=pkg DateTime::Format::MySQL

=pkg DateTime::Format::Pg


=pkgroup Hygiene

=pkg common::sense

=pkg namespace::autoclean 0.24

=pkg strictures

=pkg Perl::Tidy

=pkg Perl::Critic


=pkgroup Class bits

=pkg Class::Method::Modifiers::Fast

=pkg Class::Method::Modifiers

=pkg Reindeer 0.016

=pkg MooseX::App::Cmd

=pkg MooseX::AlwaysCoerce

=pkg MooseX::AttributeShortcuts

=pkg MooseX::AutoDestruct 0.007

=pkg MooseX::RelatedClasses

=pkg MooseX::Types::Perl

=pkg MooseX::Types::LoadableClass

=pkg MooseX::Types::Path::Class

=pkg MooseX::Types::Path::Tiny

=pkg Moo

=pkg MooX::Types::MooseLike

=pkg MooX::Cmd


=pkgroup Testing

=pkg TAP::Harness::Restricted

=pkg Test::Moose::More 0.017

=pkg Test::Routine 0.015

=pkg Test::TempDir::Tiny


=pkgroup autobox

=pkg autodie 2.22

=pkg autobox::Core

=pkg autobox::dump

=pkg autobox::JSON

=pkg autobox::Colors


=pkgroup Other Libraries I Use

=pkg aliased

=pkg lib::if::dev

=pkg opts 0.05

=pkg App::Cmd

=pkg Capture::Tiny

=pkg Carp::Always

=pkg Carp::Always::Color

=pkg Config::JFDI

=pkg File::chdir

=pkg Path::Tiny

=pkg Path::Class

=pkg Smart::Args

=pkg Smart::Comments

=pkg Sub::Install

=pkg Sub::Exporter

=pkg Try::Tiny


=pkgroup Helpers

=pkg Regexp::Debugger

=pkg L

=pkg V


=pkgroup CPAN-ish Development

=pkg Devel::Confess

=pkg Dist::Zilla::App::Command::cover

=pkg Dist::Zilla::App::Command::issues

=pkg Dist::Zilla::App::Command::gh

=pkg Dist::Zilla::App::Command::dumpwith

=pkg Dist::Zilla::PluginBundle::RSRCHBOY


=head1 SEE ALSO

Task::BeLike::RSRCHBOY::Web

=cut


Makefile.PL
#
# This file is part of Task-BeLike-RSRCHBOY
#
# This software is Copyright (c) 2011 by Chris Weyl.
#
# This is free software, licensed under:
#
#   The GNU Lesser General Public License, Version 2.1, February 1999
#
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.034.
use strict;
use warnings;

use 5.006;

use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "Modules RSRCHBOY uses!",
  "AUTHOR" => "Chris Weyl <cweyl\@alumni.drew.edu>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Task-BeLike-RSRCHBOY",
  "EXE_FILES" => [],
  "LICENSE" => "lgpl",
  "MIN_PERL_VERSION" => "5.006",
  "NAME" => "Task::BeLike::RSRCHBOY",
  "PREREQ_PM" => {
    "App::AltSQL" => 0,
    "App::Cmd" => 0,
    "App::cpanminus::reporter" => 0,
    "App::cpanoutdated" => 0,
    "App::gh" => 0,
    "CPAN::Mini" => 0,
    "CPAN::Mini::Inject" => 0,
    "Capture::Tiny" => 0,
    "Carp::Always" => 0,
    "Carp::Always::Color" => 0,
    "Class::Method::Modifiers" => 0,
    "Class::Method::Modifiers::Fast" => 0,
    "Config::JFDI" => 0,
    "DBD::SQLite" => "1.31",
    "DBIx::Class" => "0.08192",
    "DBIx::Class::Migration" => 0,
    "DBIx::Class::Schema::Loader" => "0.07010",
    "DateTime" => 0,
    "DateTime::Format::DB2" => 0,
    "DateTime::Format::DBI" => 0,
    "DateTime::Format::MySQL" => 0,
    "DateTime::Format::Pg" => 0,
    "Devel::Confess" => 0,
    "Directory::Scratch" => 0,
    "Dist::Zilla::App::Command::issues" => 0,
    "Dist::Zilla::PluginBundle::RSRCHBOY" => 0,
    "File::chdir" => 0,
    "L" => 0,
    "Moo" => 0,
    "MooX::Cmd" => 0,
    "MooX::Types::MooseLike" => 0,
    "MooseX::App::Cmd" => 0,
    "MooseX::AutoDestruct" => "0.007",
    "MooseX::RelatedClasses" => 0,
    "MooseX::Types::Perl" => 0,
    "Path::Class" => 0,
    "Perl::Critic" => 0,
    "Perl::Tidy" => 0,
    "Pod::Cpandoc" => 0,
    "Regexp::Debugger" => 0,
    "Reindeer" => "0.016",
    "Smart::Args" => 0,
    "Smart::Comments" => 0,
    "Sub::Exporter" => 0,
    "Sub::Install" => 0,
    "TAP::Harness::Restricted" => 0,
    "Task::CPAN::Reporter" => 0,
    "Test::Moose::More" => "0.017",
    "Test::Routine" => "0.015",
    "Test::TempDir::Tiny" => 0,
    "Try::Tiny" => 0,
    "V" => 0,
    "aliased" => 0,
    "autobox::Colors" => 0,
    "autobox::Core" => 0,
    "autobox::JSON" => 0,
    "autobox::dump" => 0,
    "autodie" => "2.22",
    "common::sense" => 0,
    "lib::if::dev" => 0,
    "local::lib" => 0,
    "namespace::autoclean" => 0,
    "opts" => "0.05",
    "strictures" => 0
  },
  "TEST_REQUIRES" => {
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::CheckDeps" => "0.010",
    "Test::More" => "0.94",
    "strict" => 0,
    "warnings" => 0
  },
  "VERSION" => "0.006",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "App::AltSQL" => 0,
  "App::Cmd" => 0,
  "App::cpanminus::reporter" => 0,
  "App::cpanoutdated" => 0,
  "App::gh" => 0,
  "CPAN::Mini" => 0,
  "CPAN::Mini::Inject" => 0,
  "Capture::Tiny" => 0,
  "Carp::Always" => 0,
  "Carp::Always::Color" => 0,
  "Class::Method::Modifiers" => 0,
  "Class::Method::Modifiers::Fast" => 0,
  "Config::JFDI" => 0,
  "DBD::SQLite" => "1.31",
  "DBIx::Class" => "0.08192",
  "DBIx::Class::Migration" => 0,
  "DBIx::Class::Schema::Loader" => "0.07010",
  "DateTime" => 0,
  "DateTime::Format::DB2" => 0,
  "DateTime::Format::DBI" => 0,
  "DateTime::Format::MySQL" => 0,
  "DateTime::Format::Pg" => 0,
  "Devel::Confess" => 0,
  "Directory::Scratch" => 0,
  "Dist::Zilla::App::Command::issues" => 0,
  "Dist::Zilla::PluginBundle::RSRCHBOY" => 0,
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "File::chdir" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "L" => 0,
  "Moo" => 0,
  "MooX::Cmd" => 0,
  "MooX::Types::MooseLike" => 0,
  "MooseX::App::Cmd" => 0,
  "MooseX::AutoDestruct" => "0.007",
  "MooseX::RelatedClasses" => 0,
  "MooseX::Types::Perl" => 0,
  "Path::Class" => 0,
  "Perl::Critic" => 0,
  "Perl::Tidy" => 0,
  "Pod::Cpandoc" => 0,
  "Regexp::Debugger" => 0,
  "Reindeer" => "0.016",
  "Smart::Args" => 0,
  "Smart::Comments" => 0,
  "Sub::Exporter" => 0,
  "Sub::Install" => 0,
  "TAP::Harness::Restricted" => 0,
  "Task::CPAN::Reporter" => 0,
  "Test::CheckDeps" => "0.010",
  "Test::Moose::More" => "0.017",
  "Test::More" => "0.94",
  "Test::Routine" => "0.015",
  "Test::TempDir::Tiny" => 0,
  "Try::Tiny" => 0,
  "V" => 0,
  "aliased" => 0,
  "autobox::Colors" => 0,
  "autobox::Core" => 0,
  "autobox::JSON" => 0,
  "autobox::dump" => 0,
  "autodie" => "2.22",
  "common::sense" => 0,
  "lib::if::dev" => 0,
  "local::lib" => 0,
  "namespace::autoclean" => 0,
  "opts" => "0.05",
  "strict" => 0,
  "strictures" => 0,
  "warnings" => 0
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);
